main.go

```diff
@@ -133,26 +43,49 @@ func main() {
 				log.Fatalf("couldn't close file %v, %v\n", *destFile, err)
 			}
 		}()
-
 		dst = dstFile
 	}
 
-	w := bufio.NewWriterSize(dst, *writeBufferSize)
+	data, err := mm.Map(src, mm.RDONLY, 0)
+	if err != nil {
+		log.Fatalf("couldn't mmap file: %v\n", err)
+	}
+	if len(data) <= 0 {
+		return
+	}
+
+	var str []string
 
+	// identify lines in data
+	p := 0
+	for i := 0; i < len(data); i++ {
+		if data[i] == '\n' {
+			str = append(str, sixb.BtS(data[p:i+1]))
```

Comment:
I think this implementation where newline is included into the string can be problematic when the last line of the file doesn't have the new line and it's reshuffled, then 2 lines will be printed into one line

Reply:
Yes you are right, last line also needs newline, added.
This has the advantage of less number of I/O calls when writing the file. You are making an extra call to append newline.

Reply:
Good point, including in https://github.com/afiodorov/radixmmap/pull/3